Updated recommended value for ExpectedSealDuration (#408)
filecoin-project/filecoin-docs: docs/mine/lotus/miner-configuration.md

  Filter = ""
```

> `ExpectedSealDuration` is an estimate of how long sealing will take, and is used to reject deals whose start epoch might be earlier than the expected completion of sealing. It can be estimated by [benchmarking](benchmarks.md) or by [pledging a sector](sector-pledging.md). **It's final value should be `(TIME_TO_SEAL_A_SECTOR + WaitDealsDelay) * 1.5`**. This will ensure that the miner does not commit to have the sector sealed and ready too soon when making deals.

Reviewer: ```suggestion
`ExpectedSealDuration` is an estimate of how long sealing will take, and is used to reject deals whose start epoch might be earlier than the expected completion of sealing. It can be estimated by [benchmarking](benchmarks.md) or by [pledging a sector](sector-pledging.md). 

The final value of `ExpectedSealDuration` should equal `(TIME_TO_SEAL_A_SECTOR + WaitDealsDelay) * 1.5`. This equation ensures that the miner does not commit to having the sector sealed too soon.
```

Author: @johnnymatthews This needs some extra highlighting I think.

Reviewer: ```suggestion
`ExpectedSealDuration` is an estimate of how long sealing will take, and is used to reject deals whose start epoch might be earlier than the expected completion of sealing. It can be estimated by [benchmarking](benchmarks.md) or by [pledging a sector](sector-pledging.md).

::: warning
The final value of `ExpectedSealDuration` should equal `(TIME_TO_SEAL_A_SECTOR + WaitDealsDelay) * 1.5`. This equation ensures that the miner does not commit to having the sector sealed too soon.
:::
```

How about a warning box?
